src/main/java/io/anserini/nrts/TweetServlet.java

```diff
@@ -0,0 +1,90 @@
+package io.anserini.nrts;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.http.NameValuePair;
+import org.apache.http.client.utils.URLEncodedUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.lucene.document.Document;
+import org.apache.lucene.index.DirectoryReader;
+import org.apache.lucene.index.IndexReader;
+import org.apache.lucene.queryparser.classic.ParseException;
+import org.apache.lucene.queryparser.classic.QueryParser;
+import org.apache.lucene.search.IndexSearcher;
+import org.apache.lucene.search.Query;
+import org.apache.lucene.search.ScoreDoc;
+import org.apache.lucene.search.TopScoreDocCollector;
+//import org.eclipse.jetty.http.HttpStatus;
+
+import io.anserini.nrts.TweetStreamIndexer.StatusField;
+
+public class TweetServlet extends HttpServlet {
+	private static int topN = 20;
+	private static final Logger LOG = LogManager.getLogger(TweetServlet.class);
+	private IndexReader reader;
+	
+	@Override
+	protected void doGet(HttpServletRequest request, HttpServletResponse response)
+			throws ServletException, IOException {		
+		int topN = 20;
```

Comment:
User query via a web browser can now be at either http://localhost:8080/search?query=birthday (return top 20 by default) or http://localhost:8080/search?query=birthday&top=15 (user specified).